Revert/Sources/ExampleAlerts.swift

```diff
@@ -30,25 +30,18 @@ extension UIAlertController {
   }
 }
 
-extension UIAlertView {
-  static func exampleAlertView() -> UIAlertView {
-    return UIAlertView(
-      title: NSLocalizedString("This is an UIAlertView", comment: "Alert title"),
-      message: NSLocalizedString("How are you feeling today?", comment: "Alert message"),
-      delegate: nil,
-      cancelButtonTitle: NSLocalizedString("Cancel", comment: "Alert Cancel button title"),
-      otherButtonTitles: NSLocalizedString("Ok", comment: "Alert Ok button title"),
-      NSLocalizedString("Delete", comment: "Alert Delete button title"))
-  }
-}
+// UIAlertView and UIActionSheet are not available on tvOS, so we filter out their related methods.
 
-extension UIActionSheet {
-  static func exampleActionSheet() -> UIActionSheet {
-    let actionSheet = UIActionSheet(
-      title: NSLocalizedString("This is an UIActionSheet", comment: "Alert title"),
-      delegate: nil,
-      cancelButtonTitle: NSLocalizedString("Cancel", comment: "Alert Cancel button title"),
-      destructiveButtonTitle: NSLocalizedString("Delete", comment: "Alert Delete button title"))
-    return actionSheet
+extension AlertViewController {
+  static func showExampleAlertView() {
+    #if os(iOS)
+      UIAlertView.exampleAlertView().show()
+    #endif
+  }
+  
+  static func showExampleActionsSheetInView(view : UIView) {
+    #if os(iOS)
+      UIActionSheet.exampleActionSheet().showInView(view)
+    #endif
   }
 }
```

Comment:
Same concept here than the `KeyboardHandler` below, the whole extension should only be defined on `iOS` and calls to that extension should be conditionally included.

I understand it pollutes a bit the client side of the API but calling an empty method seems confusing to me.

Have a look at `AlertViewController`. My guess would be that ideally `Identifier.AlertView` and `Identifier.ActionSheet` should only be defined on the platforms they make sense. Maybe asserting at the beginning of `displayCorrespondingAlertForIdentifier(_)` would be enough if the first option doesn't turn out to be practical.